azure-pipelines.yml

```diff
@@ -4,6 +4,7 @@ resources:
       type: github
       name: enigma-dev/RadialGM
       endpoint: github.com_EnigmaBot
+      ref: refs/pull/138/merge # Use this for testing a co-dependent rgm pr but comment this before merging
```

Comment:
reminder to remove this before merging. and to update submodule and merge RadialGM/#138 after merging this